protocol/contracts/v2/strategies/AbstractStrategy.sol

```diff
@@ -198,7 +198,7 @@ abstract contract AbstractStrategy is ITempleStrategy, TempleElevatedAccess {
     function _setTokenAllowance(IERC20 token, address spender, uint256 amount) internal {
         if (amount == token.allowance(address(this), spender)) return;
 
-        token.safeApprove(spender, 0);
+        token.approve(spender, 0);
         if (amount > 0) {
```

Comment:
Think you missed using forceApprove on this one?

Reply:
good catch!